### ISSUE TITLE
Setting Rules

### ISSUE THREAD

**Issue by mamdali96** [Leiren/Npanel#19]
I Have A set of rules that it purpose is ad-blocker and domain/ iP block 
How Can I Add Theme ?

**Comment by Leiren** [comment]
I have the domain/ip block on my plan list , but the addblock is not a priority since more management features and 
probably bug fixes should be done first.
